.idea/codeStyles/Project.xml

```diff
@@ -31,4 +29,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Please revert this file. 
Not sure what it does, but if it is on purpose, please create a new PR

Reply:
Removed the appropriate commit. Not sure, where it came from; some kind of Android Studio magic...